api/src/main/java/org/pmiops/workbench/featuredworkspace/FeaturedWorkspaceServiceImpl.java

```diff
@@ -26,9 +26,12 @@ public boolean isFeaturedWorkspace(DbWorkspace dbWorkspace) {
   }
 
   @Override
-  public FeaturedWorkspaceCategory getFeaturedCategory(DbWorkspace dbWorkspace) {
-    DbFeaturedWorkspace dbFeaturedWorkspace =
-        featuredWorkspaceDao.findByWorkspace(dbWorkspace).orElseThrow();
```

Comment:
`orElseThrow()` throws an exception that our router doesn't know about, which results in a 500, so we should avoid it where possible.  See `WorkspaceAdminServiceImpl.java` in this PR for a different way to handle this.